airbyte-webapp/src/test-utils/mock-data/mockDestinationDefinition.json

```diff
@@ -146,7 +146,12 @@
           },
           {
             "title": "verify-full",
-            "required": ["mode", "ca_certificate", "client_certificate", "client_key"],
```

Comment:
Not sure why these changed, but this update seems to be required for tests to pass

Reply:
See this slack thread for context: https://airbytehq-team.slack.com/archives/C02TXQ020QM/p1662598761408689

If you rebase off of master this should be fixed, and you shouldn't need to make any changes to this file